samples/java-protobuf-customer-registry-kafka-quickstart/pom.xml

```diff
@@ -284,9 +284,21 @@
       <scope>test</scope>
     </dependency>
     <dependency>
-      <groupId>junit</groupId>
-      <artifactId>junit</artifactId>
-      <version>4.13.2</version>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter-api</artifactId>
+      <version>5.10.1</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter</artifactId>
+      <version>5.10.1 </version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.junit.vintage</groupId>
+      <artifactId>junit-vintage-engine</artifactId>
```

Comment:
Using the vintage-engine, a can continue to use the junit4. However, the reality is that to move to junit5, a user needs to update their pom.xml anyway. So, it's a conscious decision. 

A user can eventually update the SDK version and stay on JUnit 4. If later they add a new component, the generate tests for the new component will have imports for JUnit5 and the project will fail to compile. 

Alternatively, we can add junit4, junit5 and junit-vintage-engine as transitive dependency of `kalix-java-sdk-protobuf-testkit` as such the users will always have the right dependencies on their classpath.

Reply:
btw, junit4 and junit5 are already include in the testkit, but marked as provided.